### PR TITLE
docs: add cli steps for org sync

### PR DESCRIPTION
**Pull request by EdwardAngert**
[preview](https://coder.com/docs/@15431-docs-org-sync/admin/users/idp-sync#organization-sync-premium)

### CODE REVIEW

File: docs/admin/users/idp-sync.md [hunk 1]
```diff
@@ -304,77 +339,141 @@ Visit the Coder UI to confirm these changes:
...
+You can initiate an organization sync through the CLI or through the Coder
+dashboard:
 
-> Depending on the OIDC provider, this claim may be named differently. Common
-> ones include `groups`, `memberOf`, and `roles`.
+<div class="tabs">
 
-Next configure the Coder server to read groups from the claim name with the OIDC
-organization field server flag:
+### CLI
```
Comment (Emyrk) [inline docs/admin/users/idp-sync.md:357]: Can we default to the `Dashboard` view? The cli is going to be more error prone and likely used after they understand the feature (via dashboard mucking around)

File: docs/admin/users/idp-sync.md [hunk 2]
```diff
@@ -304,77 +339,141 @@ Visit the Coder UI to confirm these changes:
...
+   | field                       | If this field is the empty string `""`, then org-sync is disabled. </br> Org memberships must be manually configured through the UI or API.                                                                                                                                             |
+   | mapping                     | Mapping takes a claim from the IdP, and associates it with 1 or more organizations by UUID. </br> No validation is done, so you can put UUID's of orgs that do not exist (a noop). The UI picker will allow selecting orgs from a drop down, and convert it to a UUID for you.          |
+   | organization_assign_default | This setting exists for maintaining backwards compatibility with single org deployments, either through their upgrade, or in perpetuity. </br> If this is set to 'true', all users will always be assigned to the default organization regardless of the mappings and their IdP claims. |
+
+### Dashboard
```
Comment (Emyrk) [inline docs/admin/users/idp-sync.md:405]:
```suggestion
### Enviroment Config
```

This reads `dashboard`, but these docs are configuring via env vars. We should change this title, and have some warning like **NOT RECOMMENDED** and **will be overwritten by database settings**.

We do need a `Dashboard` section, and that section should be the default. @jaaydenh just recently finished the UI.

**Review by Emyrk — CHANGES_REQUESTED** [review]
Content is good, just need to change section headings, and add a new `Dashboard` section, making it the default (first).

File: docs/admin/users/idp-sync.md (hunk 2 above)
Comment (EdwardAngert) [reply docs/admin/users/idp-sync.md:405]: @Emyrk I merged in the Dashboard PR, so these steps are now updated - that includes removing the env vars steps

Comment (EdwardAngert) [reply docs/admin/users/idp-sync.md]: done

**Review by Emyrk — APPROVED** [review]

**Pull request merged**